client/rpcserver/handlers.go

```diff
@@ -362,4 +385,16 @@ Returns:
       },...
     ]`,
 	},
+	registerRoute: {`"dex" "password" fee`,
```

Comment:
I'm thinking that we should adopt a convention of having the app pass be the first argument for all authenticated routes.